DSharpPlus/Net/Models/AutoModerationRuleEditModel.cs

```diff
@@ -0,0 +1,46 @@
+namespace DSharpPlus.Net.Models;
+
+using System.Collections.Generic;
+
+using DSharpPlus.Entities;
+using DSharpPlus.Enums;
+
+using Newtonsoft.Json;
+
+public class AutoModerationRuleEditModel : BaseEditModel
```

Comment:
documentation

Reply:
what can i write here? the other models don't have documentation for the global class?

Reply:
that's... worrying. i suppose it's fine as is, then